lib/engine/game/g_1848/game.rb

```diff
@@ -773,6 +806,12 @@ def check_distance(route, visits, _train = nil)
           raise GameError, 'Route must include Alice Springs'
         end
 
+        def route_distance_str(route)
```

Comment:
There's a common module used in several games you can use instead of this, see 18FL as an example

require_relative '../cities_plus_towns_route_distance_str'

Reply:
cool!